src/librustc_typeck/diagnostics.rs

```diff
@@ -4095,6 +4095,91 @@ assert_eq!(!Question::No, true);
 ```
 "##,
 
+E0604: r##"
+A cast to `char` was attempted on another type than `u8`.
```

Comment:
This reads awkwardly. I suggest "was attempted on a type other than `u8`".

This error does bring up the question of why are only `u8`s castable to `char`s? I thought a `char` in rust was 4 bytes?